packages/venly/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import { WalletInit } from '@web3-onboard/common';
+
+interface VenlyProviderOptions {
+  clientId: string
+  secretType?: any
+  environment?: string
+}
```

Comment:
```suggestion
interface VenlyProviderOptions {
  /**
  * Multiline example:
  * ClientId is a required prop and can be created at vely.com
  */
  clientId: string
  /* Single line example: secretType is optional and does this....
  secretType?: any
  environment?: string
}
```
Can you please add typescript notes to each property on what it does?

Reply:
Added typescript note for clientId.
I opted to remove secretType and environment to keep things simpler, since those can be derived from the chain property anyway.